StoreKitUnitTests/StoreKitConfigTestCase.swift

```diff
@@ -40,6 +40,16 @@ class StoreKitConfigTestCase: XCTestCase {
         userDefaults.removePersistentDomain(forName: suiteName)
     }
 
+    // - Note: Xcode throws a warning about @available and #available being redundant, but they're actually necessary:
+    // Although the method isn't supposed to be called because of our @available marks in our subclasses,
+    // everything in this class will still be called by XCTest, and it will cause errors.
+    @discardableResult func checkForiOS15APIAvailableOrSkipTest() throws -> Bool {
+        guard #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *) else {
+            throw XCTSkip("Required API is not available for this test.")
+        }
+        return true
+    }
```

Comment:
great idea!! This was driving me nuts. 

Is the boolean return needed?

Reply:
I don't think so, that was some experimentation I forgot to remove 😁